test/glific/flows/translate/open_ai_test.exs

```diff
@@ -105,4 +106,26 @@ defmodule Glific.Flows.Translate.OpenAITest do
     assert response ==
              OpenAI.check_large_strings([long_text, "thankyou for joining", "correct answer"])
   end
+
+  test "translate/3 test the possible errors" do
+    Tesla.Mock.mock(fn _env ->
+      {:error, "Mocked translation error"}
```

Comment:
@AmishaBisht can you replace it with the error we get from open ai to have better testcase